planner/core/logical_plan_builder.go

```diff
@@ -2548,12 +2548,6 @@ func tblInfoFromCol(from ast.ResultSetNode, name *types.FieldName) *model.TableI
 		if field.Name.L == name.TblName.L {
 			return field.TableInfo
 		}
-		if field.Name.L != name.TblName.L {
```

Comment:
These codes are unreachable in fact?

Reply:
Yes. I removed it because I see it's unreachable.